apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -273,11 +365,27 @@ static void tc_fs_vfs_mount(void)
 * @precondition     File system should be mounted.
 * @postcondition    NA
 */
-static void tc_fs_vfs_umount(void)
+static void tc_fs_vfs_umount_p(void)
 {
 	int ret;
+
+	tc_fs_vfs_init_mount();
+
 	ret = umount(CONFIG_MOUNT_POINT);
 	TC_ASSERT_EQ("umount", ret, OK);
+
+	TC_SUCCESS_RESULT();
+}
+
+static void tc_fs_vfs_umount_invalid_path_n(void)
+{
+	int ret;
+
+	tc_fs_vfs_init_unmount();
```

Comment:
is this correct to use 'tc_fs_vfs_init_unmount'?

Reply:
This tc is the case for unmount without any unmount path.

So to make sure that there is no mount path at initialization
The call "tc_fs_vfs_init_unmount ()" seems appropriate.

However, the name of this tc is more appropriate "tc_fs_vfs_umount_noexist_path_n".

I will modify tc name "tc_fs_vfs_umount_noexist_path_n"